governance.md

```diff
@@ -0,0 +1,107 @@
+documents, and so forth.
+
+## Maximum Representation
+
+It is the desire of the committee to limit the amount of any single employer/company over the
+project. However, currently, there is not enough diversity to make this a reality. This section
+should be revised in the future when more than one employer/company are active members of the
+Konflux community.
+
+## Committee Responsibilities and Deliverables
+
+The committee MUST:
+
+- Balance technical, architectural, and governance expertise.
+- Hold staggered terms, sufficient to ensure an orderly transition of power via elections.
+- Provide designated alternates in cases where quorum is required but not attainable with the
+  current set of members.
+
+The committee is responsible for a series of specific artifacts and activities:
+
+- The [Code of Conduct](./code_of_conduct.md) and handling violations
+- Define the processes around [ADRs](https://github.com/konflux-ci/architecture/tree/main/ADR).
+  Should the community fail to reach consensus on whether to accept a proposed ADR or not, the
+  governance committee can help break the impasse.
+
+## Governance Decision-Making Process
+
+Governance decisions are reviewed and accepted in terms of pull requests to this repository.
+
+The governance committee decisions are taken by seeking consensus towards a motion from all its
+members. If the consensus cannot be reached, the motion may be altered or dropped.
+
+Architectural decisions are handled similarly but on the
+[architecture](https://github.com/konflux-ci/architecture) repository,
+
+## Elections
+
+The election process is not yet established. It MUST be done by the next election cycle.
+
+The election process MUST also define the action required to fulfill a change in the committee
+composition. For example, when a new committee member joins, which groups should they be added to,
+and, conversely, when a member leaves, which groups should they be removed from.
+
+## Governing Committee Leave Policy
+
+Any member of the governing committee is welcome to take leave from being a governing committee
+member, for any reason (which they are not required to disclose). The actions taken will depend on
+both of:
+
+1. The length of the leave: whether this is greater or less than approximately (at the discretion of
+   the other governing committee members) 1/4 of their total term (since terms are 2 years, this
+   means the criteria is whether or not the leave is greater than 6 months).
+2. Whether their seat will be up for election during the period of their leave
+
+If the leave is less than or equal to 1/4 of the committee member's total term, and their seat will
+not be up for re-election during that time, the governing committee member may select someone to be
+an acting governing committee member for the length of the leave. This person must be eligible for
+an election even though an election will not take place.
```

Comment:
How is eligibility defined?

Reply:
To be determined. I expanded the Elections section to clarify that eligibility of voters and nominees must be clearly defined.